File: pset1/Mario.c
#include<stdio.h>
#include<cs50.h>

int main()
{
        int height,count,k;
        printf("Enter Height:");
        height=get_int();
        
        char ch='#';
        while((height<0)||(height>23))
        { 
            printf("Retry:");
             height=get_int();
             
        }
        count=height-1;
            for(int i=1;i<=height;i++)
            {
                for( k=count;k>=1;k--)
                {
                    printf("  ");
                }
                for(int j=1;j<=i;j++)
                {
                    printf("%2c",ch);
                    
                }
                printf("\n");
                count--;
            }
            return 0;
        
        
}
    

File: pset6/mario.py
import cs50

print('Height: ', end='');
h = int(input());


while(h<0 or h > 23):
   print('Height: ', end='');
   h = cs50.get_int()
   if h >= 0 and h <= 23:
            break
        

for i in range(h):
        for j in range(h - i-1):
            print(" ", end = "")
        for k in range(i+2):
            print("#", end = "")
        print("")